updater/site/github_release.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import re
+from bs4 import BeautifulSoup
+
+from updater.site.abstract_site import AbstractSite
+from updater.site.github import GitHub
+from updater.site.enum import GameVersion
+
+
+class GitHubRelease(AbstractSite):
+    _URLS = ['https://www.github.com/', 'https://github.com/']
+
+    session = requests.session()
+
+    def __init__(self, url: str):
+        super().__init__(url, GameVersion.agnostic)
+
+    @classmethod
+    def handles(cls, url: str) -> bool:
+        v = bool(re.match('^https://(www.)?github.com/[^/]+/[^/]+/releases/?$', url))
```

Comment:
redundant, but unimportant. I'll fix this before publishing a new version

Reply:
Oh you are absolutely right. I missed this when cleaning up the code.